cmd/fund/fund.go

```diff
@@ -0,0 +1,132 @@
+package fund
+
+import (
+	"crypto/ecdsa"
+	"fmt"
+	"log"
+	"math/big"
+
+	_ "embed"
+
+	"github.com/chenzhijie/go-web3"
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/crypto"
+	"github.com/spf13/cobra"
+)
+
+var (
+	//go:embed usage.md
+	usage                string
+	walletCount          int
+	fundingWalletAddress string
+	fundingWalletPK      string
+	chainID              int
+	chainRPC             string
+	walletFundingAmt     float64
+	walletFundingGas     uint64
+)
+
+func generateWalletAddresses(numWallets int) ([]*common.Address, error) {
+	var addresses []*common.Address
```

Comment:
nit: you can pre-allocate the capacity of the `addresses` slice since the size is known I think.

```suggestion
        addresses := make([]*common.Address, 0, numWallets)
```

Reply:
another good call!